libs/features/mas/src/variants/plans-v2.css.js

```diff
@@ -232,11 +260,11 @@ merch-card[variant="plans-v2"] [slot="heading-m"] span.price, merch-card[variant
 
 merch-card[variant="plans-v2"] [slot="heading-m"] span.price.price-strikethrough,
 merch-card[variant="plans-v2"] [slot="heading-m"] s {
-    font-family: 'Adobe Clean', sans-serif;
-    font-size: 18px;
-    font-weight: 400;
+    font-size: 20px;
     color: #6B6B6B;
     text-decoration: line-through;
+    font-family: 'Adobe Clean', 'adobe-clean', sans-serif;
```

Comment:
I see some inconsistencies with using `'Adobe Clean', 'adobe-clean', sans-serif` and using `'Adobe Clean', sans-serif`. Suggestion: set a variable with the first option, and use it throughout the code.